README.md

```diff
@@ -116,6 +116,9 @@ The following parameters should be set in the `driver_config` for the individual
 
 The following parameters should be set in the main `driver_config` section as they are common to all platforms:
  - `vcenter_disable_ssl_verify` - Whether or not to disable SSL verification checks. Good when using self signed certificates. Default: false
+ - `vm_wait_timeout` - How many seconds to wait for VM connectivity. Default: 90
+ - `vm_wait_interval` - Check interval between tries on VM connectivity. Default: 2.0
```

Comment:
Is the "Check interval" a length of time? A number of checks?

Reply:
It's the length of time between tries to check the VM. So basically "Number of seconds between tries"